Sources/UtiliKit/Obfuscation/ObfuscatedKey.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ObfuscatedKey.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
Looks like we need to update the copyright here to "Bottle Rocket Studios". Could you go ahead and update this comment and make the change in the project file?

Reply:
Done.